test/ripple-client-test.ts

```diff
@@ -27,11 +27,27 @@ describe('Client [Test Runner]', function () {
 
   // Collect all the tests:
   const allPublicMethods = getAllPublicMethods(new Client())
-  const allTestSuites = loadTestSuites()
+  const apiTestSuites = loadTestSuitesFromFolder("client")
 
-  // Run all the tests:
-  for (const {name: methodName, tests, config} of allTestSuites) {
-    describe(`client.${methodName}`, () => {
+  runTests(apiTestSuites, "client.")
+
```

Comment:
Why were the changes in this file necessary?

Reply:
I suppose they aren't anymore. I made it more extensible for when I had multiple folders that wanted to use the functionality and figured it didn't do harm to keep it in. (Although I also see that it's outside the scope of the task, which  makes it weird to include) Do you think I should revert it back to how it was running before?

Reply:
Yes

Reply:
Reverted